client/components/singleBoat.js

```diff
@@ -1,19 +1,28 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import { getSingleBoat } from '../store/boat';
 
 class singleBoat extends Component {
-  componentDidMount() {}
+  componentDidMount() {
+    console.log('in single boat!');
+    this.props.getSingleBoat(this.props.match.params.id);
+  }
   render() {
-    return <div />;
+    return <h2>TESTTEST</h2>;
   }
 }
 
 const mapStateToProps = state => {
-  return {};
+  console.log('mapping state to props in single boat', state);
```

Comment:
quick question --

So componentDidMount -> we can grab a single boat from database if user got here via the browser router

We can also access single boat in mapStateToProps. Do we anticipate passing in a singleBoat from another component? Is that why this is here?